apps/paf/behaviours/limit-person.js

```diff
@@ -0,0 +1,14 @@
+module.exports = superclass => class extends superclass {
+
+  locals(req, res) {
+    const locals = super.locals(req, res);
+    const persons = req.sessionModel.get('persons').aggregatedValues;
+
+    if (persons && persons.length >= 6) {
```

Comment:
Not a suggestion more a discussion

The limit is 6 right? is it worth adding a variable at the top called person limit so that it's obvious and easy to change if needs be?